sklearn/linear_model/tests/test_logistic.py

```diff
@@ -1837,15 +1861,13 @@ def test_multinomial_identifiability_on_iris(fit_intercept):
     According to [1], the maximum of the L2 penalized likelihood automatically
     satisfies the symmetric constraint:
     sum(coef_k, k=1..c) = 0
-
     Further details can be found in the appendix of [2].
 
     Reference
     ---------
     .. [1] Zhu, Ji and Trevor J. Hastie. "Classification of gene microarrays by
     penalized logistic regression". Biostatistics 5 3 (2004): 427-43.
     https://doi.org/10.1093/biostatistics%2Fkxg046
-
```

Comment:
This line should be kept (I think). We are not that consistent in scikit-learn but I think more often than not, we have blank lines between references.